src/metabase/api/automagic_dashboards.clj

```diff
@@ -101,6 +101,10 @@
   [_entity-type segment-id-str]
   (api/read-check (db/select-one Segment :id (ensure-int segment-id-str))))
 
```

Comment:
This enables the `/auto/dashboard/model/:id` endpoint (vs. question instead of model before).